test/classes/errors/nilability-assign/assign-nonnil-owned-from-oknil-nil.chpl

```diff
@@ -1,11 +0,0 @@
-//  lhs: owned!  rhs: nil?  error: nlb
-
-class MyClass {  var x: int;  }
-
-var lhs = new owned MyClass();
-
-
-lhs = nil;
-
-compilerError("done");
-
```

Comment:
Why was this test removed? I think we should keep it, as a part of a (mostly-)auto-generated suite.

Reply:
This was removed because it is no longer an error to assign a non nilable owned to nil. In discussion on the original PR offline with @bradcray we decided it was better to remove the test rather than just change the behavior of the test

Reply:
I don't think I realized that this was an auto-generated test in that discussion.  I don't remember the details of the discussion anymore, but if Vass prefers, it would be fine with me to leave this test in where, presumably, the `.good` file would be updated to show the compiler hitting the `compilerError("done")`?

Reply:
I did not realize that we can now assign nil to non-nilable owned. In this case:
* I would still like to keep the test
* Yes, the .good should be updated
* @jabraham17 please also update `test/classes/errors/nilability-generate.chpl.notest` Line 6 with this exception.